tests/e2e/vmgroups_test.go

```diff
@@ -111,6 +111,9 @@ func (vg *VmGroupTest) TearDownSuite(c *C) {
 	out, err = ssh.InvokeCommand(vg.config.EsxHost, cmd)
 	log.Printf(out)
 
+	// Remove Config DB
+	adminutils.ConfigRemove(vg.config.EsxHost)
+
```

Comment:
@govint Yesterday we (sorry you were not included) discussed this and all agreed that most of our e2e tests should be running in the "default" mode. The "default" mode here means "No Config DB", i.e. a fresh installation without running "Config Init". For now multi-tenancy feature is experimental, so (at least for now) we are expecting most of our customers will be using vDVS in the "default" mode. This assumption will remain true until multi-tenancy is fully/correctly implemented and becomes an official product feature.

Coming back to e2e tests, we are expecting most of the test groups will be run in the "default" mode without DB configured. Only vmgroup related tests will need to run "config init", and thus it is expected these tests should be responsible for cleaning up the DB (i.e. run "config rm") so that other e2e tests will still be running in a clean "default" mode.

Hope this helps. Let me know if you still have question about this.